File: Suite420-SpringBackEnd/src/main/java/br/edu/ifpb/gugawag/social/model/Cliente.java
package br.edu.ifpb.gugawag.social.model;

import javax.persistence.*;

@Entity
@Table(name = "tb_cliente")
public class Cliente {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;
    private String nome;
    private String cpf;
    private String endereco;
    private String telefone;

    public Long getId(){
        return id;
    }
    public String getNome(){
        return nome;
    }
    public void setNome(String nome){
        this.nome = nome;
    }
    public String getCpf(){
        return cpf;
    }
    public void setCpf(String cpf){
        this.cpf = cpf;
    }
    public String getEndereco(){
        return endereco;
    }
    public void setEndereco(String endereco){
        this.endereco = endereco;
    }
    public String getTelefone(){
        return telefone;
    }
    public void setTelefone(String telefone){
        this.telefone = telefone;
    }
}

File: Suite420/src/app/shared/model/cliente.ts
export class Cliente {
  id?: number;
  nome?: string;
  cpf?: string;
  endereco?: string;
  telefone?: string;

  constructor(id?: number, clientes: Cliente = {}) {
    this.id = id;
    this.nome = clientes.nome;
    this.cpf = clientes.cpf;
    this.endereco = clientes.endereco;
    this.telefone = clientes.telefone;
  }
}
